bookwyrm/activitypub/note.py

```diff
@@ -59,7 +66,8 @@ class Review(Comment):
 
 
 @dataclass(init=False)
-class Quotation(Comment):
-    ''' a quote and commentary on a book '''
-    quote: str
-    type: str = 'Quotation'
+class Rating(Comment):
+    ''' a full book review '''
+    rating: int = None
+    content: str = None
```

Comment:
I was reading up on Multi-table inheritance and see now that this removes the field from `Rating`, so ignore this comment!

Reply:
this makes the default value for content (which shouldn't be set on ratings) `None`, which means the field will be removed from the activity when it's serialized. But really it just shouldn't inherit from `Comment`, because it's silly to add the field and then hope it never gets used and cull it later instead of just not having it present. And rating shouldn't have a `None` default value.